rgbledclient.ts

```diff
@@ -0,0 +1,48 @@
+// Add your code here
+namespace jd_class {
+    export const RGB_LED = 0xe7aeb0fc;
+}
+
+const enum JDRGBLEDCommand {
+    SetColor = 0x80
+}
+
+namespace jacdac {
+    //% fixedInstances
+    export class RGBLEDClient extends Client {
+        constructor(requiredDevice: string = null) {
+            super("RGBLED", jd_class.RGB_LED, requiredDevice);
+        }
+
+        /**
+         * Set the brightness of the strip. This flag only applies to future operation.
+         * @param brightness a measure of LED brightness in 0-255. eg: 20
+         */
+        //% blockId="jdrgbled_set_brightness" block="set %strip brightness %brightness"
+        //% brightness.min=0 brightness.max=255
+        //% weight=2 blockGap=8
+        //% group="Light"
+        setBrightness(brightness: number): void {
+            this.setRegInt(REG_INTENSITY, brightness)
+        }
+
+        /**
+         * Set the colour of the RGB led in hex format.
+         * 
+         * Upper most byte is red, middle byte green, lower byte blue.
+         */
+        //% blockId="jdrgbled_set_color" block="set %strip color %colorcode"
+        //% brightness.min=0 brightness.max=255
+        //% weight=2 blockGap=8
+        //% group="RGBLED"
+        setColor(colorCode: number): void {
+            let buf = Buffer.create(4);
```

Comment:
maybe send 3 bytes, so it won't get confused if it's extended to 2 pixels?

Reply:
Okie dokie. Does the jacdac layer add the correct padding as required by the spec?